logger/redact.go

```diff
@@ -86,6 +113,29 @@ func RedactJSON(json, replacement string, keys [][]string) string {
 	return string(out)
 }
 
+func redactRecursive(obj *fastjson.Value, keys []string, replacementValue *fastjson.Value) {
+	if len(keys) == 0 {
```

Comment:
let's add null validation for `obj` since can be null

Reply:
Done in ebcf419.